src/flownet/_command_line.py

```diff
@@ -66,7 +66,26 @@ def flownet_ahm(args: argparse.Namespace) -> None:
             )
 
     config = parse_config(args.config, args.update_config)
-    run_flownet_history_matching(config, args)
+    if hasattr(args, "restart_folder") and args.restart_folder is not None:
+        if args.restart_folder.exists():
+            # check for pickled files and zipped file
+            if (
+                pathlib.Path(args.restart_folder / "network.pickled").is_file()
+                and pathlib.Path(args.restart_folder / "schedule.pickled").is_file()
+                and pathlib.Path(args.restart_folder / "parameters.pickled").is_file()
+                and pathlib.Path(
+                    args.restart_folder / "parameters_iteration-latest.parquet.gzip"
+                ).is_file()
+            ):
+                run_flownet_history_matching_from_restart(config, args)
+            else:
+                raise ValueError(
+                    f"Case in {args.restart_folder} is not complete! Some files are missing!"
```

Comment:
Maybe indicate which files are needed to thew users - makes it slightly easier to understand the problem.

Reply:
Agree. Added the file names.